aten/src/THC/generic/THCTensorMath.cu

```diff
@@ -35,35 +35,6 @@ void THCTensor_(zero)(THCState *state, THCTensor *self_)
   THCudaCheck(cudaGetLastError());
 }
 
-void THCTensor_(zerosLike)(THCState *state, THCTensor *r_, THCTensor *input)
-{
-  THCAssertSameGPU(THCTensor_(checkGPU)(state, 2, r_, input));
-  THCTensor_(resizeAs)(state, r_, input);
-  THCTensor_(zero)(state, r_);
-}
-
-void THCTensor_(onesLike)(THCState *state, THCTensor *r_, THCTensor *input)
-{
-  THCAssertSameGPU(THCTensor_(checkGPU)(state, 2, r_, input));
-  THCTensor_(resizeAs)(state, r_, input);
-  THCTensor_(fill)(state, r_, ScalarConvert<int, scalar_t>::to(1));
-}
-
-ptrdiff_t
-THCTensor_(numel)(THCState *state, THCTensor *t)
-{
-  return THCTensor_(nElement)(state, t);
-}
-
-void THCTensor_(cat)(THCState *state, THCTensor *result,
-		     THCTensor *ta, THCTensor *tb, int dimension)
-{
-  THCTensor* inputs[2];
-  inputs[0] = ta;
-  inputs[1] = tb;
-  THCTensor_(catArray)(state, result, inputs, 2, dimension);
-}
-
```

Comment:
Were there any substantive changes in this move?

Reply:
_zerosLike_ and _onesLike_ are not being used anywhere and will be killed in https://github.com/pytorch/pytorch/pull/18201

_cat_ is being used and had to be enabled for bool 
not sure about _numel_, please see discussion down below.